src/test/java/org/pentaho/metastore/test/MetaStoreFactoryTest.java

```diff
@@ -558,4 +559,16 @@ private List<DimensionAttribute> generateAttributes() {
     return list;
   }
 
+  @Test
```

Comment:
Please, use instead `@Test(expected = MetaStoreException.class)`

Reply:
Making this change results in failing test case. Needs more investigation